content/en/docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -52,6 +52,12 @@ for the [kube-apiserver](/docs/admin/kube-apiserver/), [kube-controller-manager]
 For all external cloud providers, please follow the instructions on the individual repositories,
 which are listed under their headings below, or one may view [the list of all repositories](https://github.com/kubernetes?q=cloud-provider-&type=&language=)
 
+## Alibaba Cloud
+ If you wish to use the external cloud provider, its repository is kubernetes/cloud-provider-alibaba-cloud(link:https://github.com/kubernetes/cloud-provider-alibaba-cloud)
+
+### Node Name
+The AlibabaCloudProvider use ${region-id}.${instance-id} as the name of the Kubernetes Node object which is determined by the kubelet overridden with `--hostname-override=${region-id}.${instance-id} --provider-id=${region-id}.${instance-id}`
```

Comment:
This sentence doesn't make sense to me. Can kubelet determine it or not? Is the override a must?

Reply:
After I checked it with my colleague, `provider-id` is a must, and it's better to add `--hostname-override`. ：） so keep it, that's fine.

Reply:
I agree with @tengqm that the wording here is confusing. We should explicitly call it out as a pre-requisite if that's the case.

Reply:
BTW, quote the name with `${region-id}.${instance-id}` should be better. Like the following providers, maybe we could change this sentence to:

```
The AlibabaCloudProvider ... . Note that the Kubernetes Node name must ...
```

Reply:
@zhangxiaoyu-zidif - if you can paraphrase or explain in detail what meaning you want here, I think that we can suggest revised wording.